app/views/orphans/_orphan_user.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <p class="caution"><%= t(".orphaning_bylines_only_message_html") %></p>
 
-<p class="caution">
+<p class="caution notice">
```

Comment:
The two `<p>` tags above this one are missing the extra class

Reply:
Whoops, that would be the file I tried to wrap everything in a single `<div class="caution notice">` on... I apparently misjudged how much Cmd+Z-ing to do after that didn't work out 😆